api-errors.json

```diff
@@ -253,7 +253,7 @@
     },
     "40109": {
         "short_message": "Sendrequest Or Share Already Exists",
-        "long_message": "An already existing Sendrequest exists or is blocked"
+        "long_message": "An Sendrequest or a Share already exists"
```

Comment:
Maybe the sentence should start with 'A' not 'An'?

Reply:
```suggestion
        "long_message": "A Sendrequest or Share already exists"
```

Reply:
This is very similar to the short message. Is there any other information we can add to the long message that is not in the short one?

Reply:
It's sort of messy since a sendrequest can be used for several different purposes. But I can try to add something.